---
title: "Are the non-coding regions really non-coding? Large-scale identification of orphan genes from B73 v5 genome"
subtitle: "P135"
author: "Keting Chen"
date: ""
image: "https://upload.wikimedia.org/wikipedia/commons/e/e5/ENIAC-changing_a_tube.jpg"
categories: 
- Computational and Large-Scale Biology
- Odd Numbered
freeze: true
editor: source
markdown: 
    wrap: 72
---

Orphan genes are young genes often generated by de novo gene birth from previously non-coding regions and encode proteins that have no homologs in other species. Orphan genes from diverse plant species have been experimentally demonstrated as essential for nutrient utilization, stress tolerance, or disease resistance. Orphan genes are often short in length and mono-exonic, and lack the canonical features of mature conserved genes. Hence, they are largely ignored by many gene prediction approaches. Using BIND, a gene prediction pipeline that combines ab initio prediction and evidence-based prediction utilizing ~1,500 publicly available maize B73 RNA-seq datasets from diverse biological conditions, we have identified ~38,000 transcripts that potentially encode orphan proteins. About 88% of these transcripts lie within genomic regions previously considered to be intergenic, and the remaining 12% are likely the alternative splicing products of mature genes. Approximately 1,500 orphan transcripts have translation evidence supported by analysis of 175 publicly available Ribo-seq datasets that contains only a subset of conditions of the RNA-seq datasets used to identify these orphan transcripts and thereby represents a lower bound. Three tools to evaluate coding-potential were employed to analyze the orphan transcripts, and ~39% are predicted to be protein-coding. Due to the lack of homology with other species, identifying functions of orphan genes is a major challenge. As an example strategy, we present a joint statistical analysis between the maize silk orphan transcriptome and cuticular wax metabolome. This analysis identifies ten orphans with translation evidence that are significantly associated with cuticular wax composition, thus providing novel directions for characterizing orphan function. Collectively, by identifying orphan transcripts expressing under specific conditions, this study demonstrates the complex and dynamic nature of the maize transcriptome and provides rich resources for further exploration of maize genome biology and its impact on crop phenotypes.

<a href='mailto:kchen@iastate.edu'>Chen, Keting</a><sup>1</sup><br><a href='mailto:jingli@iastate.edu'>Li, Jing</a><sup>1 2</sup><br><a href='mailto:yxguo@iastate.edu'>Guo, Yuxiang</a><sup>1 2</sup><br><a href='mailto:mash@iastate.edu'>Wurtele, Eve S.</a><sup>1 2</sup><br><a href='mailto:myn@iastate.edu'>Yandeau-Nelson, Marna D.</a><sup>1 2</sup><br>

<sup>1</sup>Department of Genetics, Development & Cell Biology, Iowa State University, Ames, Iowa, 50011<br><sup>2</sup>Bioinformatics & Computational Biology Graduate Program, Iowa State University, Ames, Iowa, 50011<br>