---
title: "Optimizing simulations for digital breeding that accurately parallels empirical data"
subtitle: "P250"
author: "Michael Burns"
date: ""
image: "https://upload.wikimedia.org/wikipedia/commons/5/5e/Aztec_Diet_Basket_of_Corn_%289755332575%29.jpg"
categories: 
- Quantitative Genetics & Breeding
- Even Numbered
freeze: true
editor: source
markdown: 
    wrap: 72
---

Computational and data-driven approaches to accelerate and optimize breeding programs are becoming a common practice among plant breeders. Simulations allow breeders to evaluate hypotheses and changes in a breeding scheme time- and cost-efficiently, but accurately simulating traits that match empirical data remains a challenge. Here, we address this problem by incorporating information about the genetic architecture (i.e., numbers of causative variants, effect sizes, and modes of gene action) from genome-wide association studies (GWAS) of common maize agronomic traits into simulations. Using at least 200 top GWAS hits as causative variants resulted in a high correlation of 0.54 between simulated and empirical trait data. In addition, reducing the original marker effect sizes estimated by GWAS models resulted in a more realistic partitioning of phenotypic variances from genotype, environment, and their interaction. Our results suggest that incorporating information from GWAS beyond statistical significance thresholds is valuable for simulating more realistic digital breeding phenotypes. To assess the utility of the simulated phenotypic data for digital breeding we are currently comparing the results of genomic prediction and machine learning models trained using the simulated data to the output of models trained on empirical data.

<a href='mailto:burns756@umn.edu'>Burns, Michael J.</a><sup>1</sup><br><a href='mailto:della028@umn.edu'>Della Coletta, Rafael</a><sup>1</sup><br><a href='mailto:samuelbf@uark.edu'>Fernandes, Samuel B.</a><sup>2</sup><br><a href='mailto:mmikel@illinois.edu'>Mikel, Mark A.</a><sup>3 4</sup><br><a href='mailto:mbohn@illinois.edu'>Bohn, Martin O.</a><sup>3</sup><br><a href='mailto:alipka@illinois.edu'>Lipka, Alexander E.</a><sup>3</sup><br><a href='mailto:cnhirsch@umn.edu'>Hirsch, Candice N.</a><sup>1</sup><br>

<sup>1</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, MN 55108, USA<br><sup>2</sup>Department of Crop, Soil, and Environmental Sciences, University of Arkansas, Fayetteville, AR 72701, USA<br><sup>3</sup>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA<br><sup>4</sup>Roy J. Carver Biotechnology Center, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA<br>