<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Presentations</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="site_libs/bootstrap/bootstrap-dark.min.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home 🌽</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_schedule.html"> 
<span class="menu-text">Schedule</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_keynotes.html"> 
<span class="menu-text">Keynotes</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./generated_talks.html" aria-current="page"> 
<span class="menu-text">Talk Abstracts</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_poster_list.html"> 
<span class="menu-text">Poster Overview</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./generated_poster_list.html"> 
<span class="menu-text">Poster Abstracts</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="./program_map.html"> 
<span class="menu-text">Map</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_general_information.html"> 
<span class="menu-text">General Information</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_awards.html"> 
<span class="menu-text">Awards</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_authors_attendees.html"> 
<span class="menu-text">Authors and Attendees</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./program_sponsors.html"> 
<span class="menu-text">Sponsors</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Presentations</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="t1-an-exocyst-component-zmexo70f4-coordinates-with-jasmonate-signaling-pathway-to-regulate-maize-immunity-to-gibberella-stalk-rot" class="level2">
<h2 class="anchored" data-anchor-id="t1-an-exocyst-component-zmexo70f4-coordinates-with-jasmonate-signaling-pathway-to-regulate-maize-immunity-to-gibberella-stalk-rot">T1: An exocyst component ZmEXO70F4 coordinates with jasmonate signaling pathway to regulate maize immunity to Gibberella stalk rot</h2>
<p><font color="#336699">Biochemical and Molecular Genetics </font> <a href="mailto:xgao@njau.edu.cn">Xiquan Gao</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:xgao@njau.edu.cn">Gao, Xiquan</a><sup>1 2 3</sup><br>Ma, Liang<sup>1 2 3</sup><br>Xie, Fugui<sup>1 2 3</sup><br>Wang, Qing<sup>1 2 3</sup><br></p>
<p><sup>1</sup>State Key Laboratory of Crop Genetics &amp; Germplasm Enhancement and Utilization, Nanjing Agricultural University, Nanjing, Jiangsu Province, 210095, P. R. China<br><sup>2</sup>College of Agriculture, Nanjing Agricultural University, Nanjing, Jiangsu Province, 210095, P. R. China<br><sup>3</sup>Collaborative Innovation Center for Modern Crop Production co-sponsored by Province and Ministry, Nanjing Agricultural University, Nanjing, Jiangsu Province, 210095, P. R. China<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-2-contents" aria-controls="callout-2" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-2" class="callout-2-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Gibberella Stalk Rot (GSR) caused by Fusarium graminearum is one of the most devastating diseases worldwide, which significantly reduces the production and quality, and restricts the application of mechanic harvesting technology in maize. Despite that several resistance genes associated with GSR have been cloned, more candidate genes need to be characterized, and the complex genetic and molecular mechanisms underlying GSR immunity await to be fully dissected, considering that the trait is controlled by many minor-effect loci. We identified a novel GSR-resistant QTN qRGS1, and ZmExo70F4 was identified as the candidate gene via Genome-wide association study (GWAS), allelic analysis, and gene functional analysis. Using yeast-two hybrid library screening, combined with RNA-seq and WGCNA, multiple immune signaling components, including those from jasmonates signaling pathways, were identified to be associated with ZmExo70F4-meidated immunity to GSR. The interactions between ZmEXO70F4 and its potential interacting proteins including ZmMYC7 was confirmed and their functions in GSR immunity were validated. ZmMYC7 was further found to modulate GSR immunity through regulating isoflavonoids production. The results obtained will not only help us to deep insights into the complex GSR resistance mechanisms, but also provide novel genetic resources for molecular breeding of maize stalk rot resistance.</p>
</div>
</div>
</div>
</section>
<section id="t2-crazy-leaves-and-truffula-trees" class="level2">
<h2 class="anchored" data-anchor-id="t2-crazy-leaves-and-truffula-trees">T2: Crazy leaves and truffula trees</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:annis.richardson@ed.ac.uk">Annis Richardson</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-3-contents" aria-controls="callout-3" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-3" class="callout-3-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Prigge, Michael J.<sup>1</sup><br>Sluis, Aaron<sup>2 3</sup><br>Morffy, Nicholas<sup>4</sup><br>de Neve, Amber<sup>2 3</sup><br>Szutu, Whitney<sup>1</sup><br>Abraham-Juárez, María Jazmín<sup>5</sup><br>Johnson, Kjel<sup>2 3</sup><br>Do, Nicole<sup>1</sup><br>Lavy, Meirav<sup>1</sup><br>Hake, Sarah<sup>2 3</sup><br>Strader, Lucia<sup>4</sup><br>Estelle, Mark<sup>1</sup><br>Richardson, Annis E<sup>2 3 6</sup><br></p>
<p><sup>1</sup>Department of Cell and Developmental Biology, School of Biological Sciences, University of California San Diego, La Jolla, CA, USA<br><sup>2</sup>USDA Plant Gene Expression Center, 800 Buchanan Street, Albany, CA 94710.<br><sup>3</sup>Department of Plant and Microbial Biology, University of California Berkeley, CA.<br><sup>4</sup>Department of Biology, Duke University, Durham, NC, USA<br><sup>5</sup>Laboratorio Nacional de Genómica para la Biodiversidad (LANGEBIO), Unidad de Genómica Avanzada, Centro de Investigación y de Estudios Avanzados (CINVESTAV), Irapuato, 36821, Mexico.<br><sup>6</sup>Institute of Molecular Plant Sciences, School of Biological Sciences, University of Edinburgh, Edinburgh, UK.<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-4-contents" aria-controls="callout-4" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-4" class="callout-4-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Plant architecture is crucial in determining productivity. In maize the iterative development of units called phytomers; composed of a node, internode, axillary meristem and organ such as a leaf; defines final plant shape. Modulation of the growth and development of each part of the phytomer also contributes to morphological diversity underpinning differences in productivity. Auxin plays a key role in regulating phytomer development, however the pathway that decodes its effects is not well understood, in part due to few reported auxin mutants with vegetative phenotypes. Here we present two dominant maize mutants with opposing vegetative phenotypes; Hoja loca1(Oja) and Truffula(Trf); both of which have defects in auxin transcriptional response. Oja plants often fail to produce leaves at each node, and those that do form can be midribless or tube-shaped. Oja is due to a mutation in the highly conserved degron motif of the auxin response repressor ZmIAA28. In contrast, Trf plants develop extra leaves at nodes, which can have multiple midribs. The Trf mutation is located in the DNA-binding domain of ZmARF28, a member of the repressive class-B Auxin Response Factor clade that we know relatively little about. Our unique comparative analysis of maize and moss, species separated by ~500 million years of evolution, reveals a novel evolutionarily conserved degron motif, shedding light on a new pathway for class-B ARF regulation in plants. Through investigating Oja and Trf we not only build new understanding of auxin-regulated phytomer development in maize, but also uncover fundamental components of auxin response regulation important for land plants.</p>
</div>
</div>
</div>
</section>
<section id="t3-a-thiamin-pyrophosphate-kinase-encoding-gene-zmtdpk2-finely-regulates-ear-length-and-grain-yield-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t3-a-thiamin-pyrophosphate-kinase-encoding-gene-zmtdpk2-finely-regulates-ear-length-and-grain-yield-in-maize">T3: A thiamin pyrophosphate kinase encoding gene ZmTDPK2 finely regulates ear length and grain yield in maize</h2>
<p><font color="#996633">Quantitative Genetics &amp; Breeding</font> <a href="mailto:1332803470@qq.com">Yun Luo</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-5-contents" aria-controls="callout-5" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-5" class="callout-5-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:1332803470@qq.com">Luo, Yun</a><sup>1</sup><br><a href="mailto:1223262917qq.com">Liu, Yu</a><sup>1</sup><br><a href="mailto:yjianbing@mail.hzau.edu.cn">Yan, Jianbing</a><sup>1 2 3</sup><br></p>
<p><sup>1</sup>National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan 430070, China<br><sup>2</sup>Hubei Hongshan Laboratory, Wuhan 430070, China<br><sup>3</sup>Yazhouwan National Laboratory, Sanya 572024, China<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-6-contents" aria-controls="callout-6" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-6" class="callout-6-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Enhancing the grain yield of maize is imperative to meet the escalating demands for maize-derived products in food, feed, and fuel. Ear length (EL) serves as a crucial determinant of grain yield and, consequently, stands as a pivotal target trait in maize breeding. However, only a limited number of EL quantitative trait loci (QTLs) have been identified and cloned, with their underlying molecular mechanisms remaining largely elusive. Here, we characterize an EL QTL, qKB6.2, to identify a maize gene controlling flower number and ear length. The qKB6.2 encodes a thiamin diphosphokinase2 (TDPK2), a gene that involves phosphorylating thiamin into thiamin pyrophosphate (TPP) in the cytoplasm. CRISPR/Cas9 knockout lines of ZmTDPK2 exhibited lower TPP levels in the developing ear, reduced florets number, EL and thus decreased grain yield. Interesting, the maize EL of ZmTDPK2 overexpression lines exhibited a trend of initially increasing and then decreasing with the rising expression levels of ZmTDPK2, suggesting that the optimal EL is manifested at the appropriate ZmTDPK2 expression levels. The favorable haplotype frequency of qKB6.2 in the association population is only 0.4%, which is rare. And, in comparison with the hybrid lines constructed with NILqkb6.2, the EL and grain yield of hybrid lines constructed with favorable haplotype NILqKB6.2, increased by 6.3%-17.4%, and 5.5%-17.4%, respectively. Moreover, foliar application of a 0.04% TPP solution on the leaf surface has been shown to elevate the yield of maize, rape and rice. Our findings suggest that ZmTDPK2 serves as a critical role in TPP-mediated female inflorescence development, EL and grain yield, and also provide a tool to improve grain productivity by optimizing ZmTDPK2 expression levels or foliar application of TPP in other cereals.</p>
</div>
</div>
</div>
</section>
<section id="t4-a-novel-domestication-gene-establishes-developmental-boundaries" class="level2">
<h2 class="anchored" data-anchor-id="t4-a-novel-domestication-gene-establishes-developmental-boundaries">T4: A novel domestication gene establishes developmental boundaries</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:georgechuck@berkeley.edu">George Chuck</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-7-contents" aria-controls="callout-7" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-7" class="callout-7-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:zbdong@cau.edu.cn">Dong, Zhaobin</a><sup>1 2</sup><br><a href="mailto:s20203010070@cau.edu.cn">Hu, Gaoyuan</a><sup>2</sup><br><a href="mailto:myrmica2@gmail.com">Shemyakina, Elena</a><sup>1</sup><br><a href="mailto:whipple@byu.edu">Whipple, Clinton</a><sup>3</sup><br><a href="mailto:georgechuck@berkeley.edu">Chuck, George</a><sup>1</sup><br></p>
<p><sup>1</sup>University of California Berkeley, Albany, CA USA 94710<br><sup>2</sup>China Agricultural University, No.&nbsp;2 Haidian District, Beijing, China<br><sup>3</sup>Brigham Young University, Provo, UT, USA 84602<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-8-contents" aria-controls="callout-8" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-8" class="callout-8-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Early farmers selected for a suite of different vegetative and reproductive crop traits during domestication but identifying the causative loci has been hampered by their epistasis and functional redundancy. Using ChIP-seq combined with genome wide association analysis, we uncovered a developmental regulator that controls both categories of traits while acting upstream of multiple domestication loci. Mapping of four maize domestication and improvement QTL identified tasselsheath4 (tsh4) as a strong candidate gene. We show that tsh4 functions redundantly with a pair of homologs to bind and activate three domestication genes, teosinte branched1, tassels replace upper ears1 and teosinte glume architecture1. tsh4 is in fact a novel domestication gene that establishes developmental boundaries and specifies meristem versus lateral organ fates. TSH4 does this by employing an unusual mechanism where it targets and represses the very same microRNAs that negatively regulate it via a double negative feedback loop. TSH4 then cements these boundaries by binding to AUX IAA genes to repress auxin responsiveness in lateral organs and redirect the flow of auxin to axillary meristems to enable their growth. One such meristem is the spikelet pair meristem that is responsible for doubling seed yield in maize versus teosinte and is missing in higher order tsh4 mutants. A comparison of the maize versus teosinte alleles of tsh4 within introgressed backgrounds revealed that the former contains a small intron deletion and is more highly expressed. Thus, by selecting for this high expressing allele early farmers were able to modify a variety of key architectural traits to maximize yield gain, placing tsh4 at the top of the domestication hierarchy.</p>
</div>
</div>
</div>
</section>
<section id="t5-leaf-angle-and-curricular-innovation-a-cure-for-research-and-undergraduate" class="level2">
<h2 class="anchored" data-anchor-id="t5-leaf-angle-and-curricular-innovation-a-cure-for-research-and-undergraduate">T5: Leaf angle and curricular innovation: a CURE for research and undergraduate</h2>
<p><font color="#993366">Education &amp; Outreach</font> <a href="mailto:mlubkowitz@smcvt.edu">Mark Lubkowitz</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-9-contents" aria-controls="callout-9" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-9" class="callout-9-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:mlubkowitz@smcvt.edu">Lubkowitz, Mark</a><sup>1</sup><br><a href="mailto:le95@cornell.edu">Evans, Lukas</a><sup>2</sup><br><a href="mailto:mjs298@cornell.edu">Scanlon, Michael J</a><sup>2</sup><br><a href="mailto:jmyu@iastate.edu">Yu, Jianming</a><sup>3</sup><br><a href="mailto:bboris@mail.smcvt.edu">Boris, Brandon N</a><sup>1</sup><br><a href="mailto:ehoffmann@mail.smcvt.edu">Hoffmann, Emilie S</a><sup>1</sup><br><a href="mailto:kkoutsoufis@mail.smcvt.edu">Koutsoufis, Kaitlyn M</a><sup>1</sup><br></p>
<p><sup>1</sup>Saint Michael's College, Colchester, VT, USA 05439<br><sup>2</sup>Plant Biology Section, School of Integrative Plant Science, Cornell University, Ithaca, NY, USA 14853<br><sup>3</sup>Department of Agronomy, Iowa State University, Ames, IA, USA 50011<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-10-contents" aria-controls="callout-10" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-10" class="callout-10-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Undergraduate research is arguably the single most important experience for inspiring students to pursue graduate school and STEM careers. Yet, most students never have the opportunity to engage in research simply because of time and limited institutional resources. Course Based Undergraduate Research Experiences (CUREs) address these issues since students and institutions are already committed to classes. CUREs are not new to undergraduate curricula but they often occur as a singular opportunity as opposed to a curricular foundation, and therefore lack the impact of a prolonged research experience. To address this issue, we have developed a four-year CURE program that consists of eleven courses spanning four STEM majors. Students experience a breadth of research systems, questions, approaches, and analyses at every developmental level and the scale of the program ensures that a significant portion of every student’s education is research based. Here we present an overview of our CURE program and then illustrate how undergraduate researchers are helping to identify transcription factors that control leaf angle in maize. Leaf angle is an important trait that has historically been manipulated by breeders to increase planting density and therefore yield. Since the auricle contributes significantly to leaf angle, undergraduates helped perform a yeast one hybrid screen using the promoters of narrowsheath1, liguleless1, liguless3, and roughsheath1; all genes that are expressed in the pre-ligular band during leaf development. Surprisingly, all of these genes were bound by some combination of only four transcription factors in the Lateral Organ Boundary (LOB) family, including ramosa2. The small number of transcription factors identified as well as the overlap suggests that these four genes are in the same regulatory network and function early in auricle development. This finding illustrates how undergraduate education, CUREs, and discovery can be joined to the benefit of all parties.</p>
</div>
</div>
</div>
</section>
<section id="t6-optimized-zmwox2a-based-transformation-system-enhances-somatic-embryogenesis-transformation-efficiency-and-gene-editing-efficiency-in-maize-zea-mays-l.-and-sorghum-sorghum-bicolor-l." class="level2">
<h2 class="anchored" data-anchor-id="t6-optimized-zmwox2a-based-transformation-system-enhances-somatic-embryogenesis-transformation-efficiency-and-gene-editing-efficiency-in-maize-zea-mays-l.-and-sorghum-sorghum-bicolor-l.">T6: Optimized ZmWOX2A-based transformation system enhances somatic embryogenesis, transformation efficiency and gene editing efficiency in maize (Zea mays L.) and sorghum (Sorghum bicolor L.)</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:matheka@wisc.edu">Jonathan Matheka</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-11-contents" aria-controls="callout-11" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-11" class="callout-11-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:matheka@wisc.edu">Matheka, Jonathan M.</a><sup>1</sup><br><a href="mailto:rcollier2@wisc.edu">Collier, Ray</a><sup>1</sup><br><a href="mailto:nwalter8888@gmail.com">Walter, Nathalie</a><sup>1</sup><br><a href="mailto:animasaun@wisc.edu">Animasaun, David</a><sup>1 3</sup><br><a href="mailto:smkaeppl@wisc.edu">Kaeppler, Shawn M.</a><sup>1 2</sup><br><a href="mailto:hfkaeppl@wisc.edu">Kaeppler, Heidi F.</a><sup>1 2</sup><br></p>
<p><sup>1</sup>Wisconsin Crop Innovation Center, University of Wisconsin, Middleton, WI, USA<br><sup>2</sup>Department of Plant and Agroecosystem Sciences, University of Wisconsin, Madison, WI, USA<br><sup>3</sup>Department of Plant Biology, University of Ilorin, Ilorin, Nigeria<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-12-contents" aria-controls="callout-12" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-12" class="callout-12-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Embryogenic cultures are key components of many maize genome engineering/editing systems. Production of embryogenic cultures is, however, highly genotype dependent. Previously, we identified an embryogenesis-related maize gene, ZmWOX2A, and demonstrated that when constitutively expressed it enabled somatic embryogenesis and regeneration of phenotypically normal plants of the recalcitrant maize genotype, B73. Initial comparisons with a BBM+WUS-based system showed that the constitutive ZmWOX2A system generally resulted in lower somatic embryogenesis and transformation frequencies (TF). To optimize the ZmWOX2A-based system, the effects of 6 terminators, 6 promoters, and 6 enhancer-containing introns were evaluated as components of the ZmWOX2A transcriptional unit (TU). Of the 6 termination sequences tested, the tobacco extensin terminator (EUt) significantly increased ZmWOX2A-induced somatic embryogenesis (SE) and was used in TUs testing promoter and intron effects. The promoters driving expression of ZmWOX2A that were tested for effects on TF and plant phenotype included constitutive (ZmUbi1p), scutella epidermis-specific (PLTPp), and 4 novel, embryo-specific promoters we recently cloned and characterized. Significantly more somatic embryos were produced per explant with the ZmUbi1p, relative to PLTPp; however, there was no significant difference in TFs between the ZmUbi1p- or PLTPp-driven ZmWOX2A/EUt constructs. Two of the four embryo-specific promoters yielded efficient TFs, but below that of either the ZmUbi1 or PLTP promoters. The highest performing embryo-specific promoter (Php20719a) was then paired in 6 intron combinations to compare effects on TF in the ZmWOX2A system. Three of the introns significantly increased TF of the Php20719ap-containing TUs more than that of the unmodified Php20719ap, two of which also outperformed PLTPp. Optimized ZmWOX2A TUs were then tested on several maize and sorghum genotypes and efficient TFs achieved in both species. Optimized vectors were also tested in maize editing experiments and resulted in efficient generation of edited, phenotypically normal plants and progeny.</p>
</div>
</div>
</div>
</section>
<section id="t7-harnessing-ai-for-agricultural-innovation" class="level2">
<h2 class="anchored" data-anchor-id="t7-harnessing-ai-for-agricultural-innovation">T7: Harnessing AI for agricultural innovation</h2>
<p><font color="#663399">Computational and Large-Scale Biology </font> <a href="mailto:steven.johnson1@bayer.com">Steve Johnson</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-13-contents" aria-controls="callout-13" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-13" class="callout-13-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:seyedvahid.mirnezami@bayer.com">Mirnezami, Sayed Vahid</a><sup>1</sup><br>Miao, Chenyong<sup>1</sup><br>Liao, Wei-Ting<sup>1</sup><br>Carpenter, Neal<sup>1</sup><br><a href="mailto:nima.hamidighalehjegh@bayer.com">Hamidi, Nima</a><sup>1</sup><br>Clark, Kelly<sup>1</sup><br><a href="mailto:steven.johnson1@bayer.com">Johnson, Steven</a><sup>1</sup><br><a href="mailto:ruth.wagner@bayer.com">Wagner, Ruth</a><sup>1</sup><br></p>
<p><sup>1</sup>Bayer Crop Science; Chesterfield, MO, 63017<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-14-contents" aria-controls="callout-14" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-14" class="callout-14-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>As the depth and complexity of scientific data and literature expand, the role of artificial intelligence (AI) has become increasingly pivotal. AI techniques are proving to be an indispensable tool for sifting through and integrating disparate streams of information, paving the way for unprecedented breakthroughs. From protein design to cell fate determination, there are a growing number of examples of AI technologies providing key contributions to scientific discovery. Bayer Crop Science is fully engaged in the pursuit of cutting-edge solutions for the next wave of maize agricultural production. To achieve this, we have adopted a multi-level strategy that harnesses the latest developments in AI technology. In this session, we will shed light on how Bayer Crop Science is leveraging AI for large-scale yield prediction, redefining phenotyping with computer vision techniques, and augmenting the productivity of individual scientists. Furthermore, we will unveil our approach to Machine Learning Operations (MLOps), which plays a crucial role in sustaining and advancing the complex array of AI models that are integral to our product development pipeline. This talk aims to provide insights into the multi-level, dynamic contributions AI can play in modern agricultural research.</p>
</div>
</div>
</div>
</section>
<section id="t8-a-practical-haplotype-graph-map-of-the-zeasynthetic-population-enables-integration-of-teosinte-alleles-into-breeding-efforts" class="level2">
<h2 class="anchored" data-anchor-id="t8-a-practical-haplotype-graph-map-of-the-zeasynthetic-population-enables-integration-of-teosinte-alleles-into-breeding-efforts">T8: A practical haplotype graph map of the ZeaSynthetic population enables integration of teosinte alleles into breeding efforts</h2>
<p><font color="#663399">Computational and Large-Scale Biology </font> <a href="mailto:mkelleher@danforthcenter.org">Micah Kelleher</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-15-contents" aria-controls="callout-15" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-15" class="callout-15-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:mkelleher@danforthcenter.org">Kelleher, Micah</a><sup>1</sup><br><a href="mailto:tkosfeld@danforthcenter.org">Kosfeld, Tim</a><sup>1</sup><br><a href="mailto:gziegler@danforthcenter.org">Ziegler, Greg</a><sup>1</sup><br><a href="mailto:cluebbert@danforthcenter.org">Luebbert, Collin</a><sup>1</sup><br><a href="mailto:m.stetter@uni-koeln.de">Stetter, Markus</a><sup>2</sup><br><a href="mailto:ibaxter@danforthcenter.org">Baxter, Ivan</a><sup>1</sup><br><a href="mailto:rossibarra@ucdavis.edu">Ross-Ibarra, Jeffrey</a><sup>3</sup><br><a href="mailto:sherry.flint-garcia@usda.gov">Flint-Garcia, Sherry</a><sup>4</sup><br></p>
<p><sup>1</sup>Donald Danforth Plant Science Center, St.&nbsp;Louis, MO 63132<br><sup>2</sup>Institute for Plant Sciences, University of Cologne, Cologne, Germany 50674<br><sup>3</sup>Department of Evolution and Ecology, University of California, Davis, CA 95616<br><sup>4</sup>United States Department of Agriculture—Agricultural Research Service Plant Genetics Research Unit, Columbia, MO 65211<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-16-contents" aria-controls="callout-16" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-16" class="callout-16-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Domestication, adaptation, and inbreeding of modern maize has resulted in a significant loss of the genetic diversity found in the wild species of maize, teosinte. Genetic variation in teosinte could provide a useful source of diversity lacking in modern maize cultivars. However, the lack of high-resolution genetic resources for the study of teosinte has made it difficult to associate that variation with agronomic traits. We produced a genetic map of the Zea Synthetic population, a genetic resource created by randomly mating a 38-accession synthetic breeding population consisting of 11 geographically diverse teosinte (Zea mays ssp. parviglumis), B73, Mo17, and the 25 inbred Nested Association Mapping (NAM) founders. After 6 generations of random mating, 2000 DHs were produced, originally to study inbreeding depression.Our approach leverages full genome sequences of the population’s parent genomes using the practical haplotype graph (PHG), a server mounted pangenome built from a SQL database populated by a reference panel of whole genome sequences. To mirror the makeup of our Zea Synthetic population, we filled a reference panel with data from the whole-genome sequences of B73, Mo17, the 25 inbred NAM founders, and 11 teosinte pseudo-genomes. Approximately 4.6 million haplotypes were sampled from individuals and groups in this diverse panel to populate the PHG with individual and consensus reference haplotypes. These haplotypes were paired with low coverage GBS sequences from the 2000 DHs to accurately impute WGS data for the entire population for use in association mapping. Our map is able to identify QTL associations for traits such as flowering time and Cadmium accumulation as well as identify regions of segregation distortion and those that have experienced selection.</p>
</div>
</div>
</div>
</section>
<section id="t9-cyto-swapping-in-maize-by-haploid-induction-with-a-cenh3-mutant" class="level2">
<h2 class="anchored" data-anchor-id="t9-cyto-swapping-in-maize-by-haploid-induction-with-a-cenh3-mutant">T9: Cyto-swapping in maize by haploid induction with a cenh3 mutant</h2>
<p><font color="#669933">Cytogenetics</font> <a href="mailto:esteban.bortiri@syngenta.com">Esteban Bortiri</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-17-contents" aria-controls="callout-17" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-17" class="callout-17-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:Esteban.Bortiri@syngenta.com">Bortiri, Esteban</a><sup>1</sup><br><a href="mailto:Becca.Selby@syngenta.com">Selby, Becca</a><sup>1</sup><br><a href="mailto:Rachel.Egger@syngenta.com">Egger, Rachel</a><sup>1</sup><br><a href="mailto:lindsey.tolhurst@gmail.com">Tolhurst, Lindsey</a><sup>2</sup><br><a href="mailto:Shujie.Dong@syngenta.com">Dong, Shujie</a><sup>1</sup><br><a href="mailto:Kayla.Beam@syngenta.com">Beam, Kayla</a><sup>1</sup><br><a href="mailto:Kerry.Meier@syngenta.com">Meier, Kerry</a><sup>1</sup><br><a href="mailto:Jonathan.Fabish@syngenta.com">Fabish, Jonathan</a><sup>1</sup><br><a href="mailto:Mary.Dunn@syngenta.com">Dunn, Mary</a><sup>1</sup><br><a href="mailto:Dawn.McNamara@syngenta.com">McNamara, Dawn</a><sup>1</sup><br><a href="mailto:Kristin.Setliff@syngenta.com">Setliff, Kristin</a><sup>1</sup><br><a href="mailto:Rafaela_Castro_Miranda.Lunny@syngenta.com">Castro Miranda Lunny, Rafaela</a><sup>1</sup><br><a href="mailto:Donna.Delaney@syngenta.com">Delaney, Donna</a><sup>3</sup><br><a href="mailto:Sharon.Gergen@syngenta.com">Gergen, Sharon</a><sup>4</sup><br><a href="mailto:kdawe@uga.edu">Dawe, R. Kelly</a><sup>5</sup><br><a href="mailto:Tim.Kelliher@syngenta.com">Kelliher, Tim</a><sup>1</sup><br></p>
<p><sup>1</sup>Syngenta Crop Protection, 9 Davis Drive, Research Triangle Park, North Carolina 27709, USA<br><sup>2</sup>Greenlight Biosciences, 9 Laboratory Drive, Research Triangle Park, North Carolina 27709, USA<br><sup>3</sup>Syngenta Seeds, 2369 330th Street, Slater, Iowa 50244, USA<br><sup>4</sup>Syngenta Seeds, 317 330th Street, Stanton, Minnesota 55018, USA<br><sup>5</sup>Department of Genetics, University of Georgia, Athens, GA 30602, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-18-contents" aria-controls="callout-18" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-18" class="callout-18-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Modern agriculture is increasingly reliant on traited hybrid maize because of its superior yield, resistance to insect damage, and ease of weed control. Detasseling of female line plants during commercial seed production costs millions of dollars annually and lowers seed yield. The use of Cytoplasmic Male Sterility (CMS) avoids detasseling and decreases the cost of hybrid seed. However, conversion of female maize lines to CMS involves several rounds of backcrosses after the initial cross to a CMS donor line to reconstitute the traited female Recurrent Parent (RP) with minimal linkage drag. This process can take up to 2 years and results in approximately 95% Recurrent Parent (RP) recovery. Maize mutants of the centromeric histone H3 (cenH3) gene can form haploids that inherit only chromosomes of the pollinating parent but the cytoplasm from the female parent. We successfully used a cenH3 mutant to cyto-swap seven Non-Stiff Stalk elite lines by using them as pollen donors to cenH3 mutant ears, identifying haploids by genotyping, and doubling their genome. The rate of haploids recovered was 5% and genotyping with a large panel of markers confirmed the Double Haploids are 100% RP. We created a cenH3 mutant line that carries the dominant embryo color marker R1-scm2 and a CMS-C cytoplasm to perform conversions to CMS by cyto-swapping. This line was used to convert nine Stiff Stalk and four sweet corn lines to CMS in under 10 months, averaging 5% of haploids in the F1. The lines converted to CMS by cyto-swapping are 100% isogenic to RP and have no genome from the conversion line. The conversion line has no transgenic elements and has been declared exempt from regulation status by the USDA. We envision that cyto-swapping with cenH3 will be applicable in CMS deployment in diverse crops to create superior hybrid varieties.</p>
</div>
</div>
</div>
</section>
<section id="t10-a-foundational-large-language-model-for-edible-plant-genomes-provides-accurate-predictions-of-regulatory-elements-and-gene-expression-in-the-maize-genome" class="level2">
<h2 class="anchored" data-anchor-id="t10-a-foundational-large-language-model-for-edible-plant-genomes-provides-accurate-predictions-of-regulatory-elements-and-gene-expression-in-the-maize-genome">T10: A foundational large language model for edible plant genomes provides accurate predictions of regulatory elements and gene expression in the maize genome</h2>
<p><font color="#663399">Computational and Large-Scale Biology </font> <a href="mailto:j.mendoza@instadeep.com">Javier Mendoza Revilla</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-19-contents" aria-controls="callout-19" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-19" class="callout-19-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Mendoza Revilla, Javier<sup>1</sup><br>Trop, Evan<sup>1</sup><br>Gonzalez, Liam<sup>1</sup><br>Roller, Masa<sup>1</sup><br>Dalla-Torre, Hugo<sup>1</sup><br>de Almeida, Bernardo P<sup>1</sup><br>Richard, Guillaume<sup>1</sup><br>Caton, Jonathan<sup>2</sup><br>Lopez Carranza, Nicolas<sup>1</sup><br>Skwark, Marcin<sup>1</sup><br>Laterre, Alex<sup>1</sup><br>Beguir, Karim<sup>1</sup><br>Pierrot, Thomas<sup>1</sup><br>Lopez, Marie<sup>1</sup><br></p>
<p><sup>1</sup>InstaDeep<br><sup>2</sup>Google DeepMind<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-20-contents" aria-controls="callout-20" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-20" class="callout-20-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Significant progress has been made in the field of plant genomics in recent years, especially through the increased use of high-throughput methodologies that characterize multiple genome-wide molecular phenotypes. Given the time and cost of performing these experiments, a critical step for more efficient crop improvement is effectively leveraging them to make accurate predictions from genomic sequences alone. We present a novel foundational large language model, the Agronomic Nucleotide Transformer (AgroNT), based on the transformer architecture and pre-trained on reference genomes from 48 edible plant species. To explore how the AgroNT performs on predictions from genomic sequences alone, we further fine-tune it with genome-wide high-throughput experiments and compare performance to other available models. In multiple prediction tasks and across diverse plant species, we show that AgroNT obtains state-of-the-art predictions for many genomic elements in multiple plant genomes. In maize, we apply the AgroNT to multiple prediction tasks related to gene regulation. We have shown it has high performance (AUC = 0.99) in classifying maize transcripts as protein-coding (mRNA) or potentially regulatory (lncRNAs). Leveraging Assay for Transposase-Accessible Chromatin with high-throughput sequencing (ATAC-seq), it also classifies chromatin regions as open with high accuracy (AUC = 0.98). Next, we explored AgroNT’s ability to make numerical predictions from promoter sequences. First, we predict promoter strength as measured by Self-Transcribing Active Regulatory Region sequencing (STARR-seq) and obtain high correlation between observed and predicted values (average R2 0.73). We also use in vivo RNA-seq experiments to predict gene expression levels across 23 diverse maize tissues and show that our predictions correlate well with experimentally measured values (average R2 0.62). These results show that our foundational large language model has learned functional genomic features and can accelerate trait research via discovery of gene regulatory features from sequence alone, potentially accelerating efforts in genomic improvement.</p>
</div>
</div>
</div>
</section>
<section id="t11-biological-nitrogen-gixation-on-the-aerial-roots-of-maize-for-sustainable-agriculture" class="level2">
<h2 class="anchored" data-anchor-id="t11-biological-nitrogen-gixation-on-the-aerial-roots-of-maize-for-sustainable-agriculture">T11: Biological nitrogen gixation on the aerial roots of maize for sustainable agriculture</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:jeanmichel.ane@wisc.edu">Jean-Michel Ane</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-21-contents" aria-controls="callout-21" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-21" class="callout-21-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:kgibson8@wisc.edu">Gibson, Kimberly</a><sup>1</sup><br><a href="mailto:espejelvenad@wisc.edu">Venado, Rafael E.</a><sup>1</sup><br><a href="mailto:j.wilker@cgiar.org">Wilker, Jennifer</a><sup>1</sup><br><a href="mailto:daniel.laspisa@uga.edu">Laspisa, Daniel</a><sup>2</sup><br><a href="mailto:vinfante@wisc.edu">Infante, Valentina</a><sup>1</sup><br><a href="mailto:cicalderon@wisc.edu">Calderon, Claudia I.</a><sup>3</sup><br><a href="mailto:farobbins@wisc.edu">Robbins, Fletcher</a><sup>1</sup><br><a href="mailto:farobbins@wisc.edu">Walace, Jason</a><sup>2</sup><br><a href="mailto:jeanmichel.ane@wisc.edu">Ane, Jean-Michel</a><sup>1 3</sup><br></p>
<p><sup>1</sup>Department of Bacteriology. University of Wisconsin-Madison, Madison, Wisconsin, USA<br><sup>2</sup>Department of Crop &amp; Soil Sciences, University of Georgia, Georgia, USA<br><sup>3</sup>Department of Plant and Agroecosystem Sciences, University of Wisconsin-Madison, Madison, Wisconsin, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-22-contents" aria-controls="callout-22" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-22" class="callout-22-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Maize (Zea mays L.) is an important crop due to its high yield and versatility, contributing significantly to global food security and economic stability. We have shown that some maize landraces from southern Mexico obtain 29–82% of their nitrogen from the atmosphere by hosting nitrogen-fixing (diazotrophic) communities in a mucilage produced by their aerial roots after rain. The maize mucilage is produced by border cells that detach from the aerial root cap in a process similar to the underground root cap. However, border cells are much more abundant and larger in the mucilage from aerial roots than those produced by underground roots. We characterized the aerial root development of several maize landraces and showed the effect of both genetic and environmental factors. A quantitative trait loci approach was conducted in biparental and doubled haploid populations to identify plant genes controlling the number of nodes with aerial roots and their thickness. Humidity and soil nitrogen also affect aerial root development. We successfully isolated over 150 distinct diazotrophs from maize mucilage, developed non-fixing mutants as controls, and enhanced ammonium-excreting mutants from some of these isolates. The distinct symbiotic relationship of diazotrophs with maize aerial roots offers a promising avenue for utilizing biological nitrogen fixation to reduce reliance on synthetic fertilizers and promote more productive and sustainable agricultural practices.</p>
</div>
</div>
</div>
</section>
<section id="t12-decoding-nitrogen-use-efficiency-in-maize-and-sorghum-insights-from-comparative-gene-regulatory-networks-for-sustainable-agriculture" class="level2">
<h2 class="anchored" data-anchor-id="t12-decoding-nitrogen-use-efficiency-in-maize-and-sorghum-insights-from-comparative-gene-regulatory-networks-for-sustainable-agriculture">T12: Decoding nitrogen use efficiency in maize and sorghum: Insights from comparative gene regulatory networks for sustainable agriculture</h2>
<p><font color="#663399">Computational and Large-Scale Biology </font> <a href="mailto:braynen@cshl.edu">Janeen Braynen</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-23-contents" aria-controls="callout-23" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-23" class="callout-23-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:braynen@cshl.edu">Braynen, Janeen</a><sup>1</sup><br><a href="mailto:zhangl@cshl.edu">Zhang, Lifang</a><sup>1</sup><br><a href="mailto:kumari@cshl.edu">Kumari, Sunita</a><sup>1</sup><br><a href="mailto:olson@cshl.edu">Olson, Andrew</a><sup>1</sup><br><a href="mailto:vkumar@cshl.edu">Kumar, Vivek</a><sup>1</sup><br><a href="mailto:regulski@cshl.edu">Regulski, Michael</a><sup>1</sup><br><a href="mailto:cliseron21@gmail.com">Liseron-Monfils, Christophe</a><sup>2</sup><br><a href="mailto:leon.kochian@gifs.ca">Kochian, Leon</a><sup>3</sup><br><a href="mailto:bo.shen@corteva.com">Shen, Bo</a><sup>4</sup><br><a href="mailto:mary.frank@corteva.com">Frank, Mary</a><sup>4</sup><br><a href="mailto:shane.abbitt@corteva.com">Abbitt, Shane</a><sup>4</sup><br><a href="mailto:agaudinier@ucdavis.edu">Gaudinier, Allison</a><sup>5</sup><br><a href="mailto:sbrady@ucdavis.edu">Brady, Siohban</a><sup>6</sup><br><a href="mailto:ware@cshl.edu">Ware, Doreen</a><sup>1 7</sup><br></p>
<p><sup>1</sup>Cold Spring Harbor Laboratory, 1 Bungtown Road, Cold Spring Harbor, NY 11724<br><sup>2</sup>National Research Council Canada / Conseil national de recherches Canada, 1200 Montreal Road Ottawa, Ontario K1A 0R6, CA<br><sup>3</sup>Miller Institute University of California, Berkeley, CA 94720<br><sup>4</sup>Corteva, Agriscience.<br><sup>5</sup>University of Saskatchewan, Saskatoon, SK S7N4L8, CA<br><sup>6</sup>Department of Plant Biology and Genome Center, UC Davis, Davis, CA, 95616<br><sup>7</sup>USDA-ARS-NAA, Ithaca, NY 1485<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-24-contents" aria-controls="callout-24" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-24" class="callout-24-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The dual role of nitrogen in plant growth is essential yet potentially harmful in excess, posing challenges for agricultural productivity and environmental sustainability. Understanding plant adaptation to varying nitrogen levels is crucial for developing crops with improved nitrogen use efficiency (NUE). This study aimed to: 1) identify conserved regulatory architectures between dicots and monocots using Yeast One-Hybrid (Y1H) Gene Regulatory Networks (GRNs), and 2) characterize subnetworks within these GRNs related to nitrogen limitation and recovery in maize and sorghum. We constructed a maize-specific GRN, comprising 1625 Protein-DNA Interactions (PDIs) with 70 promoters and 301 transcription factors (TFs), and compared it to an existing Arabidopsis GRN. Our analysis revealed that 18% of the conserved interactions in the Arabidopsis GRN were matched by 11% in maize, indicating weaker conservation among transporter genes but significant conservation in the nitrate assimilation pathway (38 PDIs in Arabidopsis, 65 PDIs in maize). The bZIP family of transcription factors emerged as key regulators in maize, especially bZIP18/bZIP30, which demonstrated substantial outdegree interactions and feed-forward loops. Under hydroponic conditions with varying ammonium nitrate concentrations, maize and sorghum plants were grown and their leaf and root tissues sampled for RNA-seq analysis. Both species exhibited variable proportions of differentially expressed genes (DEGs) across different time points and tissues. In maize, DEGs peaked at 38% in leaves after 3 hours and 27% in roots during the 24.5-hour recovery phase. Sorghum displayed 35% of genes differentially expressed in leaves at 3 hours and 27% in roots at 24.5 hours. Among various transcription factors, maize and sorghum showed divergent expression patterns, particularly within the bZIP family and NIN-LIKE PROTEIN (NLP) transcription factors known for nitrogen regulation. This study illuminates complex genomic regulatory frameworks underpinning plant adaptability to fluctuating nitrogen levels, providing valuable insights for selecting genotypes with enhanced NUE, contributing to sustainable agricultural practices. This project was funded by the USDA-ARS award number 8062-21000-044-000D.</p>
</div>
</div>
</div>
</section>
<section id="t13-the-nitrogen-cycle-in-the-rhizosphere-of-diverse-grass-species-in-andropogoneae" class="level2">
<h2 class="anchored" data-anchor-id="t13-the-nitrogen-cycle-in-the-rhizosphere-of-diverse-grass-species-in-andropogoneae">T13: The nitrogen cycle in the rhizosphere of diverse grass species in Andropogoneae</h2>
<p><font color="#FF0066">Evolution and Population Genetics</font> <a href="mailto:sh2246@cornell.edu">Sheng-Kai Hsu</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-25-contents" aria-controls="callout-25" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-25" class="callout-25-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:sh2246@cornell.edu">Hsu, Sheng-Kai</a><sup>1</sup><br>Emmett, Bryan<sup>2</sup><br>Costa-Neto, Germano<sup>1</sup><br>Hale, Charles<sup>3</sup><br>Schulz, Aimee J<sup>3</sup><br>Lepak, Nick<sup>4</sup><br>La, Thuy<sup>1</sup><br>AuBuchon-Elder, Taylor<sup>5</sup><br>Kellogg, Elizabeth A<sup>5</sup><br>Raglin, Sierra S<sup>6</sup><br>Kent, Angela D<sup>6 7</sup><br>Romay, Cinta M<sup>1</sup><br>Buckler, Edward S<sup>1 3 4</sup><br></p>
<p><sup>1</sup>Institute for Genomic Diversity, Cornell University, Ithaca, NY, USA 14853<br><sup>2</sup>USDA-ARS, National Laboratory for Agriculture and the Environment, Ames, IA, USA 50011<br><sup>3</sup>Section of Plant Breeding and Genetics, Cornell University, Ithaca, NY USA 14853<br><sup>4</sup>USDA-ARS, Robert W. Holley Center for Agriculture and Health, Ithaca, NY, USA 14853<br><sup>5</sup>Donald Danforth Plant Science Center, St.&nbsp;Louis, MO USA 63132<br><sup>6</sup>University of Illinois at Urbana-Champaign Institute for Genomic Biology, Urbana, IL, USA 61801<br><sup>7</sup>Department of Natural Resources and Environmental Sciences, University of Illinois at Urbana-Champaign, Urbana, IL, USA 61801<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-26-contents" aria-controls="callout-26" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-26" class="callout-26-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Nitrogen (N) fertilization in modern crop production has a profound impact on the N cycle in the ecosystem. Excess N in the soil leads to nitrous oxide (N2O) emissions and to nitrate (NO3-) leaching. Breeding for efficient N use and recycling in a cropping system is crucial for global sustainability. An optimized strategy for N reuse may not exist in modern crops given the lack of selection and strong bottleneck during domestication. However, we hypothesize their wild relatives may keep the favorable phenotypes. Hence, we examined the N cycle in the rhizosphere of 39 grass species related to maize and sorghum. We discovered significant phylogenetic variation in potential nitrification rates and NO3- loss in the rhizosphere soil of diverse species. Contrary to our hypothesis, all annual species including sorghum and maize are amongst the N “Conservationists,” inhibiting nitrification and conserving N within the system. Many perennials are either “Leachers” that enhanced nitrification and leaching or “Nitrate Keepers” with similar nitrification enhancing effect but lost less NO3-. Although this result rejects our hypothesis, it unveils the diverse abilities to influence soil microbial chemistry and the varying N use efficiencies of grass species. We investigated the native habitat environments of the studied species and showed that several soil characteristics including acidity and fertility have been the key drivers shaping the plant-microbe interaction in the rhizosphere. Using comparative genomic and transcriptomic approaches, we nominated key genes encoding transporters for inorganic N and secondary metabolites that underlies the phylogenetic variation in rhizosphere N cycle. We anticipate these genes could be harnessed to improve agricultural sustainability with further functional studies.</p>
</div>
</div>
</div>
</section>
<section id="t14-genomic-structural-variation-underlies-differential-root-responses-to-nitrogen-stress-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t14-genomic-structural-variation-underlies-differential-root-responses-to-nitrogen-stress-in-maize">T14: Genomic structural variation underlies differential root responses to nitrogen stress in maize</h2>
<p><font color="#996633">Quantitative Genetics &amp; Breeding</font> <a href="mailto:spklein@iastate.edu">Stephanie Klein</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-27-contents" aria-controls="callout-27" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-27" class="callout-27-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:spklein@iastate.edu">Klein, Stephanie P.</a><sup>1</sup><br><a href="mailto:bacharya@iastate.edu">Acharya, Bimala</a><sup>1</sup><br><a href="mailto:kaitlinj@iastate.edu">Higgins, Kaitlin</a><sup>1</sup><br><a href="mailto:arowzee@iastate.edu">Rowzee, Anna</a><sup>1</sup><br><a href="mailto:ellatown@iastate.edu">Townsend, Ella</a><sup>1</sup><br><a href="mailto:connerv@iastate.edu">Valentine, Conner</a><sup>1</sup><br><a href="mailto:sna@iastate.edu">Anderson, Sarah</a><sup>1</sup><br></p>
<p><sup>1</sup>Department of Genetics, Developmental &amp; Cell Biology; Iowa State University; Ames, IA, USA, 50011<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-28-contents" aria-controls="callout-28" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-28" class="callout-28-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Uncovering the functional relationships between genome and phenome are critical for efficiently selecting desirable traits for plant resilience to environmental stress. Unlike genomic prediction that uses markers to link genotype to phenotype, whole genome assemblies can be used to predict phenotypic outcomes based on structural variation in genic and the vast intergenic regions. Using the maize Nested Association Mapping (NAM) population, we sought to identify structural variants associated with root growth angle (RGA), a root architectural trait that is important for nitrogen (N) acquisition and plastic in low N environments. Gene candidates contributing to RGA have been recently detected, but the genetic control of its plasticity is poorly understood. We surveyed the NAM founders in the field under high and low N and identified two genotypes that warranted further investigation because of their contrasting root responses to low N: B73 (steep-angled and non-responsive to stress) and Oh7B (whose RGA became significantly steeper under low N). In a followup field study, we observed a wide range of RGA plasticity in response to low N in the NAM family of recombinant inbred lines derived from B73 and Oh7B. Using linkage mapping, we identified genomic regions associated with RGA plasticity and evaluated these regions further by developing a tool to compare genomic structural variation between B73 and Oh7B. To bridge the gap between phenotypes and genome regions, we also assessed differential transcript abundance within these regions using RNAseq to identify specific elements that could account for these differential phenotypic responses to low N. Our results demonstrate that we can identify new breeding targets for increased resiliency by using a multiomics approach.</p>
</div>
</div>
</div>
</section>
<section id="t15-mining-ancient-genomes-for-microbiome-associated-phenotypes-for-nutrient-retention-in-agroecosystems" class="level2">
<h2 class="anchored" data-anchor-id="t15-mining-ancient-genomes-for-microbiome-associated-phenotypes-for-nutrient-retention-in-agroecosystems">T15: Mining ancient genomes for microbiome associated phenotypes for nutrient retention in agroecosystems</h2>
<p><font color="#996633">Quantitative Genetics &amp; Breeding</font> <a href="mailto:akent@illinois.edu">Angela Kent</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-29-contents" aria-controls="callout-29" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-29" class="callout-29-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:akent@illinois.edu">Kent, Angela</a><sup>1</sup><br><a href="mailto:favelaa1@uci.edu">Favela, Alonso</a><sup>2</sup><br><a href="mailto:mbohn@illinois.edu">Bohn, Martin</a><sup>3</sup><br></p>
<p><sup>1</sup>Department of Natural Resources and Environmental Sciences, University of Illinois at Urbana-Champaign, Urbanal IL USA 61801<br><sup>2</sup>School of Plant Science, University of Arizona, Tucson AZ USA 85721<br><sup>3</sup>Department of Crop Sciences, University of Illinois at Urbana-Champaign, Urbanal IL USA 61801<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-30-contents" aria-controls="callout-30" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-30" class="callout-30-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Plant-microbe interactions in the rhizosphere govern availability of nutrients and are a target for improving crop sustainability. However, the ability of modern crops to recruit and structure their microbiome has been altered by domestication and breeding. Selection for crop plants based on aboveground traits in high nutrient environments has inadvertently led to changes to belowground plant physiology and relationships with the soil microbiome, altering microbiome functions that contribute to sustainability and environmental quality (e.g.&nbsp;nutrient acquisition, nutrient retention, and GHG production). We hypothesized that plant genotypes differ in their ability to recruit microbial functional groups, and ultimately that the functional profile of the microbial community can be treated as a selectable plant phenotype and optimized through plant breeding. We surveyed the rhizosphere microbiome of diverse maize genotypes to compare their capability to recruit microbial nitrogen cycling functional groups and examined rates of nitrogen transformations. Significantly different N cycling microbial communities were observed among maize genotypes that represent the endpoints of directed evolution, as well as among germplasm selected under different levels of fertilization. Our results showed significantly lower rates of nitrification and denitrification in ancestral lineages of maize (teosinte). Maize-teosinte near isogenic lines were used to narrow down the genetic region associated with these nutrient retention traits to explore the mechanistic basis for these microbiome-associated phenotypes. We explored the combinatorial effect of plant-microbe interactions for their ability to both provision (via N fixation) and retain nutrients, which resulted in elevated nutrient acquisition by the crop as well as sustainable environmental outcomes. Our results link host-associated microbiome and ecosystem function, and demonstrate the genetic capacity to optimize recruitment of N-cycling functional groups and improve crop sustainability. Identifying microbiome-associated phenotypes will allow breeders and ecosystem scientists to select crop cultivars that improve the efficiency and sustainability of agriculture and protect environmental quality.</p>
</div>
</div>
</div>
</section>
<section id="t16-teosinte-pollen-drive-guides-maize-diversification-and-domestication-by-rnai" class="level2">
<h2 class="anchored" data-anchor-id="t16-teosinte-pollen-drive-guides-maize-diversification-and-domestication-by-rnai">T16: Teosinte pollen drive guides maize diversification and domestication by RNAi</h2>
<p><font color="#0066CC">Transposons &amp; Epigenetics </font> <a href="mailto:martiens@cshl.edu">Rob Martienssen</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-31-contents" aria-controls="callout-31" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-31" class="callout-31-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:bberube@ohalogenetics.com">Berube, Benjamin</a><sup>1</sup><br><a href="mailto:eernst@cshl.edu">Ernst, Evan</a><sup>1</sup><br><a href="mailto:jcahn@cshl.edu">Cahn, Jonathan</a><sup>1</sup><br><a href="mailto:benjamin.roche@und.edu">Roche, Benjamin</a><sup>1</sup><br><a href="mailto:cdesantis@cshl.edu">de Santis Alves, Cristiane</a><sup>1</sup><br><a href="mailto:jlynn@cshl.edu">Lynn, Jason</a><sup>1</sup><br><a href="mailto:scheben@cshl.edu">Scheben, Armin</a><sup>2</sup><br><a href="mailto:asiepel@cshl.edu">Siepel, Adam</a><sup>2</sup><br><a href="mailto:rossibarra@ucdavis.edu">Ross-Ibarra, Jeffrey</a><sup>3</sup><br><a href="mailto:kermicle@wisc.edu">Kermicle, Jerry</a><sup>4</sup><br><a href="mailto:martiens@cshl.edu">Martienssen, Robert</a><sup>1</sup><br></p>
<p><sup>1</sup>Howard Hughes Medical Institute, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724<br><sup>2</sup>Simons Center for Quantitative Biology, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724<br><sup>3</sup>Dept. of Evolution &amp; Ecology, Center for Population Biology and Genome Center, University of California, Davis, CA<br><sup>4</sup>Laboratory of Genetics, University of Wisconsin, Madison, WI<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-32-contents" aria-controls="callout-32" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-32" class="callout-32-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Meiotic drivers subvert Mendelian expectations by manipulating reproductive development to bias their own transmission. Chromosomal drive typically functions in asymmetric female meiosis, while gene drive is normally postmeiotic and typically found in males. Using single molecule and single-pollen genome sequencing, we describe Teosinte Pollen Drive, an instance of gene drive in hybrids between maize (Zea mays ssp. mays) and teosinte mexicana (Zea mays ssp. mexicana), that depends on RNA interference (RNAi). 22nt small RNAs from a non-coding RNA hairpin in mexicana depend on Dicer-Like 2 (Dcl2) and target Teosinte Drive Responder 1 (Tdr1), which encodes a lipase required for pollen viability. Dcl2, Tdr1, and the hairpin are in tight pseudolinkage on chromosome 5, but only when transmitted through the male. Introgression of mexicana into early cultivated maize is thought to have been critical to its geographical dispersal throughout the Americas, and a tightly linked inversion in mexicana spans a major domestication sweep in modern maize. A survey of maize landraces and sympatric populations of teosinte mexicana reveals correlated patterns of admixture among unlinked genes required for RNAi on at least 4 chromosomes that are also subject to gene drive in pollen from synthetic hybrids. Teosinte Pollen Drive likely played a major role in maize domestication and diversification, and offers an explanation for the widespread abundance of “self” small RNAs in the germlines of plants and animals.</p>
</div>
</div>
</div>
</section>
<section id="t17-how-recombination-facilitated-the-domestication-of-maize." class="level2">
<h2 class="anchored" data-anchor-id="t17-how-recombination-facilitated-the-domestication-of-maize.">T17: How recombination facilitated the domestication of maize.</h2>
<p><font color="#FF0066">Evolution and Population Genetics</font> <a href="mailto:rke27@cornell.edu">Ruth Epstein</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-33-contents" aria-controls="callout-33" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-33" class="callout-33-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:rke27@cornell.edu">Epstein, Ruth</a><sup>1</sup><br><a href="mailto:jjwheeler4120@gmail.com">Wheeler, JJ</a><sup>1</sup><br><a href="mailto:mt269@cornell.edu">Hubisz, Melissa</a><sup>2</sup><br><a href="mailto:qisun@cornell.edu">Sun, Qi</a><sup>2</sup><br><a href="mailto:bukowski@cornell.edu">Bukowski, Robert</a><sup>1 2</sup><br><a href="mailto:wp45@cornell.edu">Pawlowski, Wojtek</a><sup>1</sup><br></p>
<p><sup>1</sup>Section of Plant Breeding, School of Integrative Plant Science, Cornell University<br><sup>2</sup>Bioinformatics Facility, Cornell University<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-34-contents" aria-controls="callout-34" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-34" class="callout-34-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Since maize was domesticated 10,000 years ago, its genome composition has been radically altered. Although selection during domestication reduced the genetic diversity of maize, meiotic recombination counteracted this reduction through the breakage of linkages between loci. Therefore, recombination has played a major role in shaping the genomic landscape of modern maize. It has been hypothesized that domestication increases global recombination rates. However, the role of recombination during domestication has not been thoroughly investigated and the intricate relationship between the recombination landscape and maize domestication has not been examined. Studying this phenomenon on a fine scale requires large high-resolution recombination maps, which do not exist in plants. To explore the maize recombination landscape during domestication, we inferred past recombination events through the population genomics method of coalescence, which is a backward-in-time approach to reconstruct a population’s genealogy. We also used an identity-by-descent analysis, which identifies shared haplotype blocks. Together, these approaches yielded one million recombination events enabling us to construct the first high-resolution maps of recombination rates in plants. These unparalleled maps permitted us to examine the variation between the maize and teosinte recombination landscapes. First, we were able to confirm domestication increased the global recombination rate in maize (0.71 cM/Mb) compared to teosinte (0.63 cM/Mb). We also found that maize recombination rates were additionally increased at the boundaries of domestication selective sweeps, suggesting that recombination increased the efficiency of selection. To understand mechanistically how recombination rates evolved, we investigated recombination genes in maize and teosinte. We found selection signatures in recombination genes in maize and identified some of those genes to be either candidates for direct selection targets or located within domestication selective sweeps. We propose that selection during domestication affected recombination patterns which in turn made selection more efficient.</p>
</div>
</div>
</div>
</section>
<section id="t18-maize-development-is-controlled-by-an-ancient-nutrient-signaling-pathway" class="level2">
<h2 class="anchored" data-anchor-id="t18-maize-development-is-controlled-by-an-ancient-nutrient-signaling-pathway">T18: Maize development is controlled by an ancient nutrient signaling pathway</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:busche2@wisc.edu">Michael Busche</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-35-contents" aria-controls="callout-35" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-35" class="callout-35-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:busche2@wisc.edu">Busche, Michael</a><sup>1</sup><br><a href="mailto:hake@berkeley.edu">Hake, Sarah</a><sup>2</sup><br><a href="mailto:brunkard@wisc.edu">Brunkard, Jacob O.</a><sup>1</sup><br></p>
<p><sup>1</sup>University of Wisconsin, Madison; Madison, WI USA 53703<br><sup>2</sup>Plant Gene Expression Center, USDA-ARS and UC Berkeley; Albany CA 94710<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-36-contents" aria-controls="callout-36" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-36" class="callout-36-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The rate of development in maize is stable under many stresses but its tempo changes significantly when nutrients, including nitrogen or carbohydrates, are limiting. TARGET OF RAPAMYCIN (TOR) is a deeply conserved eukaryotic protein kinase that monitors cellular resources to coordinate growth and development with the availability of these essential nutrients. To study the impact of changing nutrient availability on maize development, we designed a nutrient-limiting assay with sixteen unique conditions to precisely toggle nutrient-sensitive developmental responses in growing maize embryos. We performed functional genomic experiments including transcriptomics, quantitative proteomics and phosphoproteomics, metabolomics, polysome profiling, and Ribo-seq on embryos grown under these conditions. Combined with phenotypic and targeted molecular analyses, these experiments demonstrate that TOR precisely responds to changes in sugar availability by regulating gene expression at every level—transcriptionally, post-transcriptionally, translationally, and post-translationally. We have identified thousands of DEGs, hundreds differentially abundant metabolites, and dozens of phosphorylation events all dependent on TOR’s response to nutrient levels. More specifically, we have identified deeply conserved residues in TOR targets that are identical across 150 million years of evolution. We have also identified novel targets and networks in maize related to C4 photosynthesis and grass developmental patterning. Together, these data provide a map for future genetic interventions to enhance nutrient use efficiency in maize.</p>
</div>
</div>
</div>
</section>
<section id="t19-fractionation-in-the-tripsacinae-subtribe" class="level2">
<h2 class="anchored" data-anchor-id="t19-fractionation-in-the-tripsacinae-subtribe">T19: Fractionation in the tripsacinae subtribe</h2>
<p><font color="#FF0066">Evolution and Population Genetics</font> <a href="mailto:snodgras@iastate.edu">Samantha Snodgrass</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-37-contents" aria-controls="callout-37" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-37" class="callout-37-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:snodgras@iastate.edu">Snodgrass, Samantha J.</a><sup>1</sup><br><a href="mailto:margaret.woodhouse@usda.gov">Woodhouse, Margaret R.</a><sup>2</sup><br><a href="mailto:arn@iastate.edu">Seetharam, Arun S.</a><sup>1 3</sup><br><a href="mailto:mcs368@cornell.edu">Stitzer, Michelle C.</a><sup>4</sup><br><a href="mailto:mhufford@iastate.edu">Hufford, Matthew B.</a><sup>1</sup><br></p>
<p><sup>1</sup>Department of Ecology, Evolution, and Organismal Biology, Iowa State University, Ames, IA, 50010<br><sup>2</sup>USDA-ARS, Corn Insects and Crop Genetics Research Unit, Ames, IA, 50010<br><sup>3</sup>Department of Genetics, Development, and Cellular Biology, Iowa State University, Ames, IA, 50010<br><sup>4</sup>Institute for Genomic Diversity, Cornell University, Ithaca, NY, 14853<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-38-contents" aria-controls="callout-38" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-38" class="callout-38-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Polyploidy and fractionation— the processes by which genomes grow and shrink in size—characterize plant genome evolution. The rate of fractionation has typically been studied using a single species relative to a diploid reference or a set of species each descendent from different whole genome duplication events (WGD). From these, fractionation has been hypothesized to occur quickly after the initial WGD event and the genome re-diploidized before subsequent speciation events. The ancestral species of the Tripsacinae subtribe, which contains the genera Tripsacum and Zea, underwent a WGD ~12MYA after divergence from Sorghum. The descendant species that comprise the subtribe, including maize, have since fractionated back to a diploid-like state with Zea genomes comprised of 10 chromosomes and Tripsacum 18. Previous work in maize demonstrate ongoing fractionation at the timescale of hundreds of years, much more recent than earlier hypothesized. To better estimate the timing and rate of fractionation, we use high-quality, de novo assemblies of genomes across the Tripsacinae. We developed a pipeline to accurately identify shared and segregating fractionation of homoeologs across 35 Tripsacinae genomes. Using parsimony, the timing of fractionation is estimated across the clade. Preliminary findings suggest that, while most fractionation is shared across all genomes and occurred before the divergence of genera or right after, Tripsacum shows more retention of homoeologs than Zea, indicating that the rate of fractionation varies between lineages. Substantial chromosomal rearrangements and downsizing appear to uniquely characterize Zea species. By using a wide sampling of genomes descending from a single WGD event, we gain a better understanding of the complex and dynamic processes of genome evolution after polyploidy and provide a framework for future exploration of other ancient polyploids.</p>
</div>
</div>
</div>
</section>
<section id="t20-zmira1-a-new-root-system-architecture-gene-identified-by-phenomics" class="level2">
<h2 class="anchored" data-anchor-id="t20-zmira1-a-new-root-system-architecture-gene-identified-by-phenomics">T20: ZmIRA1, a new root system architecture gene identified by phenomics</h2>
<p><font color="#996633">Quantitative Genetics &amp; Breeding</font> <a href="mailto:michelle.s.cho@wustl.edu">Michelle Cho</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-39-contents" aria-controls="callout-39" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-39" class="callout-39-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:michelle.s.cho@wustl.edu">Cho, Michelle</a><sup>1 2</sup><br>Liu, Zhengbin<sup>3</sup><br>Yang, Bing<sup>2 4</sup><br>Gunn, Shayla<sup>2</sup><br>Duncan, Keith<sup>2</sup><br>Sellers, Mitchell<sup>2</sup><br>Lebow, Clara<sup>2</sup><br>Moose, Stephen<sup>5</sup><br>Topp, Christopher<sup>2</sup><br></p>
<p><sup>1</sup>Division of Biology and Biomedical Studies, Department of Biology, Washington University in St.&nbsp;Louis, St.&nbsp;Louis, MO, 63110<br><sup>2</sup>Donald Danforth Plant Science Center, St.&nbsp;Louis, MO, 63132 USA<br><sup>3</sup>Partek Inc, Chesterfield, MO, 63005 USA<br><sup>4</sup>Division of Plant Sciences, Bond Life Sciences Center, University of Missouri, Columbia, MO 65211, USA.<br><sup>5</sup>Department of Crop Sciences, University of Illinois Urbana-Champaign, Urbana, IL 61801, USA.<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-40-contents" aria-controls="callout-40" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-40" class="callout-40-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Despite roots being a major organ essential to the well-being and survival of the plant the inherent difficulties in phenotyping them have resulted in inadequate knowledge of their genetic control. Recent techniques developed in our lab have allowed high-throughput quantification of roots in 2D and 3D to find genetic factors that regulate root system architecture (RSA). We applied these tools to the Illinois Long Term Protein Selection Strains (ILTPS) that have been recurrently selected for &gt;100 years for their high and low protein content in the kernel, hypothesizing that the direct selection of nitrogen content in kernels also led to indirect selection of root phenotypes involved in nitrogen uptake. Using the recombinant inbred population of the ILTPS, the Illinois Protein Strain Recombinant Inbreds (IPSRIs), we performed a genome-wide association study (GWAS) with root traits extracted from 2D images of field-grown maize. A gene that we will refer to as Ideal Root Architecture 1 (ZmIRA1) was identified and is thought to function in central carbon-nitrogen metabolism. ZmIRA1 was confirmed to have promoter indels and differential expression between Illinois High Protein (IHP) and Illinois Low Protein (ILP) inbred lines, defining two alleles. In field experiments conducted in 2020 and 2022 with the IPSRI population, field-grown root crowns were measured from X-ray tomography-generated models and a custom feature extraction pipeline. We found that possessing either the IHP or ILP allele for ZmIRA1 was correlated with phenotypic differences in maize RSA. Subsequent examination of ZmIRA1 was done by CRISPR-Cas9 knockout mutants in a B73 background. In a greenhouse experiment with high nitrogen, we found that the mutants have specific root system architectural differences compared to the WT validating the potential link between central metabolism and root growth.</p>
</div>
</div>
</div>
</section>
<section id="t21-genetic-and-environmental-impacts-on-the-maize-microbiome-across-the-united-states" class="level2">
<h2 class="anchored" data-anchor-id="t21-genetic-and-environmental-impacts-on-the-maize-microbiome-across-the-united-states">T21: Genetic and environmental impacts on the maize microbiome across the United States</h2>
<p><font color="#996633">Quantitative Genetics &amp; Breeding</font> <a href="mailto:jason.wallace@uga.edu">Jason Wallace</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-41-contents" aria-controls="callout-41" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-41" class="callout-41-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:jason.wallace@uga.edu">Wallace, Jason G.</a><sup>1 2 3</sup><br><a href="mailto:hl46161@uga.edu">Li, Hanxia</a><sup>1</sup><br><a href="mailto:corey.schultz09@gmail.com">Schultz, Corey</a><sup>1</sup><br><a href="mailto:Hanish.Desai@uga.edu">Desai, Hanish</a><sup>4</sup><br></p>
<p><sup>1</sup>Department of Crop &amp; Soil Sciences, University of Georgia, Athens, Georgia, USA 30602<br><sup>2</sup>Institute of Plant Breeding, Genetics, &amp; Genomics, University of Georgia, Athens, Georgia, USA 30602<br><sup>3</sup>Institute of Bioinformatics, University of Georgia, Athens, Georgia, USA 30602<br><sup>4</sup>Biological Sciences, University of Georgia, Athens, Georgia, USA 30602<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-42-contents" aria-controls="callout-42" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-42" class="callout-42-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Microbes live on, around, and inside maize, yet we know relatively little about what most of them are doing and how they impact the plant. Two key questions are to what degree the local environment and host genetics shape these microbial communities, and what sort of interactions happen between them. To address these questions, we performed two analyses of maize microbiomes across the US as part of the Genomes to Fields Initiative. Our first analysis focused on soil and endophyte microbiomes from young (</p>
</div>
</div>
</div>
</section>
<section id="t22-exploring-the-cis-regulatory-landscape-of-the-b73-and-mo17-genomes" class="level2">
<h2 class="anchored" data-anchor-id="t22-exploring-the-cis-regulatory-landscape-of-the-b73-and-mo17-genomes">T22: Exploring the cis-regulatory landscape of the B73 and Mo17 genomes</h2>
<p><font color="#663399">Computational and Large-Scale Biology </font> <a href="mailto:agallavotti@waksman.rutgers.edu">Andrea Gallavotti</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-43-contents" aria-controls="callout-43" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-43" class="callout-43-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:marygalli@waksman.rutgers.edu">Galli, Mary</a><sup>1</sup><br><a href="mailto:zlchen@waksman.rutgers.edu">Chen, Zongliang</a><sup>1</sup><br><a href="mailto:tg2210@nyu.edu">Ghandour, Tara</a><sup>2</sup><br><a href="mailto:jrg233@scarletmail.rutgers.edu">Gregory, Jason</a><sup>1</sup><br><a href="mailto:amina.chaudhry@waksman.rutgers.edu">Chaudhry, Amina</a><sup>1</sup><br><a href="mailto:XuanZhang@uga.edu">Zhang, Xuan</a><sup>3</sup><br><a href="mailto:schmitz@uga.edu">Schmitz, Robert</a><sup>3</sup><br><a href="mailto:ch153@nyu.edu">Huang, S. Carol</a><sup>2</sup><br><a href="mailto:agallavotti@waksman.rutgers.edu">Gallavotti, Andrea</a><sup>1 4</sup><br></p>
<p><sup>1</sup>Waksman Institute of Microbiology, Rutgers University, Piscataway, NJ 08854-8020,, USA<br><sup>2</sup>New York University, New York, NY 10003, USA<br><sup>3</sup>University of Georgia, Athens, GA 30602, USA<br><sup>4</sup>Department of Plant Biology, Rutgers University, New Brunswick, NJ 08901, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-44-contents" aria-controls="callout-44" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-44" class="callout-44-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>The domestication and improvement of many plant species frequently involved modulation of transcriptional outputs and continues to offer much promise for targeted trait engineering. The cis-regulatory regions controlling trait-associated transcriptional variants however reside within non-coding regions that are currently poorly annotated. This is particularly true in large crop genomes such as maize where regulatory regions constitute only a small fraction of the total genomic space. The current understanding of transcriptional regulation in eukaryotic organisms with large genomes posits that transcription factors (TFs) bind to short DNA sequence motifs (TF binding sites or TFBSs) in small regional clusters of accessible chromatin called cis-regulatory modules (CRMs), and control the gene expression changes responsible for plant developmental programs and environmental responses. CRMs can be in proximity of core promoters, upstream and downstream of the genes they regulate, and can even be located tens or hundreds of kilobases away from their regulated targets, complicating their identification. While sequences matching TFBS motifs are highly abundant within a genome, only a very small fraction is actually bound by TFs and affect gene expression. To improve the characterization of the functional regulatory space in maize, we used DAP-seq (DNA-affinity purification sequencing) to profile binding sites for ~200 TFs belonging to 30 distinct TF families in both B73 and Mo17 genomes. This allowed us to capture the DNA binding landscape of distinct TF families, and to assemble a panel of 66 TFs that represent a large portion of TFBS diversity. Overlap with orthogonal datasets of accessible chromatin regions uncovered the portion of the genome occupied by functional CRMs. Comparative analysis of TF binding in the two genomes revealed extensive variation in position, organization and structures of CRMs; the majority of this diversity is caused by existing structural variation. Guided by high-resolution maps of proximal and distal CRMs, we are disrupting specific CRMs and measuring phenotypic outcomes, with a focus on genes regulating plant architecture and biotic stresses. Our approach identifies functional CRMs and ultimately aids in understanding how TF-DNA binding and its variability in different genetic backgrounds affects gene regulation and ultimately phenotypic outcomes in maize.</p>
</div>
</div>
</div>
</section>
<section id="t23-understanding-the-role-of-zmwus1-and-cis-regulatory-elements-in-maize-inflorescence-development-at-single-cell-resolution" class="level2">
<h2 class="anchored" data-anchor-id="t23-understanding-the-role-of-zmwus1-and-cis-regulatory-elements-in-maize-inflorescence-development-at-single-cell-resolution">T23: Understanding the role of ZmWUS1 and cis-regulatory elements in maize inflorescence development at single-cell resolution</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:sohyunbk@gmail.com">Sohyun Bang</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-45-contents" aria-controls="callout-45" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-45" class="callout-45-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:sohyunbk@gmail.com">Bang, Sohyun</a><sup>1</sup><br><a href="mailto:xuanzhang@uga.edu">Zhang, Xuan</a><sup>2</sup><br><a href="mailto:jrg233@scarletmail.rutgers.edu">Gregory, Jason</a><sup>3</sup><br><a href="mailto:zlchen@waksman.rutgers.edu">Chen, Zongliang</a><sup>3</sup><br><a href="mailto:marygalli@waksman.rutgers.edu">Galli, Mary</a><sup>3</sup><br><a href="mailto:agallavotti@waksman.rutgers.edu">Gallavotti, Andrea</a><sup>3</sup><br><a href="mailto:schmitz@uga.edu">Schmitz, Robert J.</a><sup>2</sup><br></p>
<p><sup>1</sup>Institute of Bioinformatics, University of Georgia, Athens, GA, 30602, USA<br><sup>2</sup>Department of Genetics, University of Georgia, Athens, GA 30602, USA<br><sup>3</sup>Waksman Institute of Microbiology, Rutgers University, Piscataway, NJ 08854-8020, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-46-contents" aria-controls="callout-46" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-46" class="callout-46-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Maize inflorescences arise through cell divisions from a group of undifferentiated stem cells within the inflorescence meristem. The organization of stem cells in the meristems is maintained by WUSCHEL (WUS) in the organizing center, which is situated beneath the stem cells. WUS encodes a homeodomain transcription factor whose expression is regulated by a negative feedback loop with CLAVATA3. The Barren inflorescence3 (Bif3) mutant in maize is caused by a tandem duplicated copy of ZmWUS1, and it displays phenotypic alterations in ear, enlarged inflorescences and reduced numbers of spikelet-pair meristems. We hypothesized that ZmWUS1 overexpression alters gene expression by modifying cis-regulatory elements (CREs) near target genes. The precise target region of WUS remains elusive due to restricted expression and activity of WUS in specific cells. Here, we performed an in vivo approach, single-cell Assay for Transposase-Accessible Chromatin with sequencing (scATAC-seq) in early developing ears in wild type and Bif3 mutants to identify CREs affected by overexpressed ZmWUS1. We identified 14 distinct cell types, including organizing center with high chromatin accessibility around ZmWUS1 correlating with its known expression. Using the ZmWUS1-bound motif TGAATGAA, obtained in DNA Affinity Purification and sequencing (DAP-seq) experiments, we found ZmWUS1binding activity in outer cells, encompassing stem cells. This suggests ZmWUS1 migration from the organizing center to outer cells. However, alternative CREs were observed in the organizing center of Bif3, but not in outer cells. This indicates that in Bif3, excess ZmWUS1 binds non-target CREs in the organizing center, leading to altered gene regulation. Further investigation revealed increased chromatin accessibility in CREs of the Bif3 organizing center, enriched with the motif TWAT, distinct from the one identified in DAP-seq. These findings show that altered ZmWUS1 levels influence its binding behavior in the organizing center of Bif3, and may explain the drastic phenotypic changes observed in mutant ears.</p>
</div>
</div>
</div>
</section>
<section id="t24-integrating-single-cell-transcriptomes-links-organ-specific-gene-expression-with-function" class="level2">
<h2 class="anchored" data-anchor-id="t24-integrating-single-cell-transcriptomes-links-organ-specific-gene-expression-with-function">T24: Integrating single-cell transcriptomes links organ-specific gene expression with function</h2>
<p><font color="#339966">Cell and Developmental Biology </font> <a href="mailto:kenchanmane@gmail.com">Sunil Kenchanmane Raju</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-47-contents" aria-controls="callout-47" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-47" class="callout-47-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:kenchanmane@gmail.com">Kenchanmane Raju, Sunil</a><sup>1</sup><br><a href="mailto:bg93@nyu.edu">Guillotin, Bruno</a><sup>1</sup><br><a href="mailto:hailongyang@umass.edu">Yang, Hailong</a><sup>2</sup><br><a href="mailto:pli1@nyu.edu">Ip, Pui-Leng</a><sup>1</sup><br><a href="mailto:mbartlett@umass.edu">Bartlett, Madelaine</a><sup>2</sup><br><a href="mailto:ken.birnbaum@nyu.edu">Birnbaum, Ken</a><sup>1</sup><br></p>
<p><sup>1</sup>Center for Genomics and Systems Biology, New York University, New York, NY, USA<br><sup>2</sup>Biology Department, University of Massachusetts, Amherst, MA, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-48-contents" aria-controls="callout-48" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-48" class="callout-48-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Root architecture determines spatiotemporal control of resource exploration and acquisition, leading to crucial implications in plant productivity. Little is known about cell type differences among these roots at the molecular level and how these affect cellular organization, development, and function. Here, we present a comparative study of single-cell transcriptomes from the embryonic primary and seminal roots and post-embryonic lateral and crown roots in maize B73. Integration of scRNAseq datasets combining more than 24,000 cells revealed conserved and unique transcriptional programs in cell type specification and allowed dissection of gene expression pattern changes within cell types. While primary and secondary roots morphologically differ in their number of cortical layers, we didn’t see significant differences in the number of cells with cortical identity. However, a significantly higher number of cells showed endodermis identity in lateral roots, revealing a potential root-type specific transcriptional specialization. Post-embryonic crown roots showed an overrepresentation of genes involved in water transport, epidermis differentiation, and response to stress compared to primary and seminal roots. We also identified genes that switched cell-type-specific expression patterns across root types. For example, grassy tillers1, a class I HD-Zip gene that controls lateral branching, showed broad expression across all cell types in the lateral roots, while its expression was restricted to specific cell types such as pericycle and less differentiated xylem and phloem in primary and seminal roots, and almost absent in crown roots. The gt1 mutant phenotype consistently shows longer lateral roots, suggesting that gt1’s growth suppression phenotypes are conserved in shoot and root. These results together show the potential of scRNAseq in unraveling functionally relevant molecular complexity of root architecture in maize.</p>
</div>
</div>
</div>
</section>
<section id="t25-meiotic-recombination-differs-between-sexes-in-maize." class="level2">
<h2 class="anchored" data-anchor-id="t25-meiotic-recombination-differs-between-sexes-in-maize.">T25: Meiotic recombination differs between sexes in maize.</h2>
<p><font color="#336699">Biochemical and Molecular Genetics </font> <a href="mailto:lee.gwonjin@ufl.edu">Gwonjin Lee</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-49-contents" aria-controls="callout-49" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-49" class="callout-49-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:lee.gwonjin@ufl.edu">Lee, Gwonjin</a><sup>1</sup><br><a href="mailto:meixiazhao@ufl.edu">Zhao, Meixia</a><sup>1</sup><br></p>
<p><sup>1</sup>Department of Microbiology and Cell Science, University of Florida, Gainesville, FL 32611<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-50-contents" aria-controls="callout-50" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-50" class="callout-50-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Meiotic recombination is a fundamental biological process that exchanges genetic material between homologous chromosomes during meiosis. It involves the physical swapping of DNA strands, known as crossover (CO), which influences genetic diversity in offspring. However, a comprehensive understanding of the patterns and regulation of COs, especially in female and male meiocytes in maize, remains to be elucidated. In this study, we used parental inbred lines from the maize nested association mapping (NAM) populations, representing more than 85% of intraspecies genetic diversity in maize, to explore variations and factors involved in meiotic recombination between the sexes. We performed genotyping-by-sequencing (GBS) on approximately 3,000 backcrossed (BC1) individuals derived from 15 male and female F1 hybrids. Some BC1 hybrids showed significantly more COs in either males or females, particularly with higher recombination rates at the chromosome ends. In other lines, no significant differences were observed between sexes in COs at the whole-genome level, although some distinctions were found at the chromosome level. These results suggest that meiotic recombination differs considerably among different maize genetic backgrounds. Furthermore, CO frequencies in females were negatively correlated with CHH (where H = A, T, or C) methylation levels in somatic cells, indicating that standing variation in CHH methylation could impact meiotic recombination variability in female meiocytes across different lines. These observations, together with further investigations into genetic and epigenetic factors, may contribute to a comprehensive understanding of the regulation of sex-specific meiotic recombination.</p>
</div>
</div>
</div>
</section>
<section id="t26-x1-encodes-a-putative-rna-binding-protein-required-for-paramutations-at-multiple-loci" class="level2">
<h2 class="anchored" data-anchor-id="t26-x1-encodes-a-putative-rna-binding-protein-required-for-paramutations-at-multiple-loci">T26: x1 encodes a putative RNA-binding protein required for paramutations at multiple loci</h2>
<p><font color="#0066CC">Transposons &amp; Epigenetics </font> <a href="mailto:marchus.3@buckeyemail.osu.edu">Oliver Marchus</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-51-contents" aria-controls="callout-51" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-51" class="callout-51-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:marchus.3@buckeyemail.osu.edu">Marchus, Oliver J</a><sup>1</sup><br><a href="mailto:hollick.3@osu.edu">Hollick, Jay B</a><sup>1</sup><br></p>
<p><sup>1</sup>Department of Molecular Genetics and Centers for Applied Plant Sciences and RNA Biology, The Ohio State University, Columbus, OH, 43210<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-52-contents" aria-controls="callout-52" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-52" class="callout-52-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Paramutations are defined by trans-homolog interactions resulting in meiotically-heritable epigenetic changes affecting gene regulation. While the nature of these interactions remains unclear, an Arabidopsis thaliana (A.t.)-based 24 nucleotide (nt) RNA-directed DNA methylation (RdDM)-like mechanism provides one working model. In maize, paramutations occur between specific alleles at several loci including purple plant1 (pl1) and booster1 (b1) which encode transcription factors promoting purple/red pigmentation. Relative to highly-expressed reference states (Pl-Rh and B-I), paramutant derivatives (Pl′ and B′ ) confer weak plant color and are exclusively transmitted from respective Pl-Rh / Pl′ and B-I / B′ heterozygotes in apparent violation of Mendelian segregation. In forward genetic screens for factors required to maintain repression (rmr) of Pl′ states, ethyl methanesulfonate-induced mutations define at least sixteen loci. All seven known RMR proteins facilitate RNA polymerase IV-derived 24nt RNA biogenesis, yet no RdDM-like 24nt RNA effector proteins have been identified. Here we show the rmr17 locus is genetically required for paramutations at both pl1 and b1. Bulk-segregant WGS analysis and deletion-based validation map rmr17 to the x1 gene located within the highly-studied a1–sh2 interval. x1 encodes the founding member of a protein family with namesake XS domains responsible for binding double-stranded (ds) RNA. The likely A.t. X1 orthologs represent a clade of three non-redundant paralogs forming heterotetrameric complexes required for RdDM. These complexes 1) bind duplexes of 24nt RNA and RNA polymerase V-derived long non-coding RNA (lncRNA) scaffolds, 2) interact with the SWI3B nucleosome remodeler, and 3) help recruit the de novo cytosine methyltransferase DRM2. Our findings support a role for X1 in paramutation by recognizing 24nt RNA-lncRNA duplexes and recruiting proteins that facilitate meiotically-heritable repressive chromatin modifications. The identification and characterization of rmr17 thus represents a significant advance in understanding the role of 24nt RNAs in defining heritable epigenetic variations.</p>
</div>
</div>
</div>
</section>
<section id="t27-strigolactone-control-of-iron-homeostasis-in-maize." class="level2">
<h2 class="anchored" data-anchor-id="t27-strigolactone-control-of-iron-homeostasis-in-maize.">T27: Strigolactone control of iron homeostasis in maize.</h2>
<p><font color="#336699">Biochemical and Molecular Genetics </font> <a href="mailto:ewalker@umass.edu">Elsbeth Walker</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-53-contents" aria-controls="callout-53" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-53" class="callout-53-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:ewalker@umass.edu">Walker, Elsbeth L</a><sup>1</sup><br><a href="mailto:sfili@umass.edu">Fili, Stavroula</a><sup>1 2</sup><br><a href="mailto:guanjc@ufl.edu">Guan, Jiahn-Chou</a><sup>3</sup><br><a href="mailto:kekoch@ufl.edu">Koch, Karen E.</a><sup>3 4 5</sup><br></p>
<p><sup>1</sup>Biology Department, University of Massachusetts, Amherst; Amherst, MA 01003<br><sup>2</sup>Plant Biology Graduate Program, University of Massachusetts, Amherst; Amherst, MA 01003<br><sup>3</sup>Horticultural Sciences Department, University of Florida; Gainesville, Florida 32611<br><sup>4</sup>Plant Molecular Biology Program, University of Florida; Gainesville. Florida 32611<br><sup>5</sup>3.5. Genetics Institute, University of Florida; Gainesville, Florida, 32610<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-54-contents" aria-controls="callout-54" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-54" class="callout-54-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Strigolactones are plant hormones with roles in a wide range of signaling and developmental processes. A yellow-striped maize mutant, (interveinal yellow) ivy, was determined to have low iron in tissues under normal growth conditions. The gene underlying the ivy mutation was mapped and identified as ZmCCD8, a key enzyme in the biosynthesis of strigolactones. Under iron-replete conditions, comparison of the transcriptomes of wild-type plants and maize ccd8 mutants revealed suppression of several iron-regulated genes in ccd8. These genes are normally up-regulated during iron deficiency and include the key iron-regulated transcription factor IRO2 as well as genes involved in the biosynthesis of iron chelators and transporters. Additionally, external supply of synthetic strigolactone in the mutants alleviated chlorosis and returned iron-regulated gene expression to wild-type levels. In iron limited conditions, iron-regulated gene expression in ccd8 mutants responded normally, indicating that strigolactones are not required for the iron deficiency response. However, they are required for basal expression of iron-regulated genes when adequate iron is available, highlighting a distinction between iron homeostasis during normal growth, and the iron deficiency response. The connection between strigolactones and iron homeostasis is not limited to maize, as Arabidopisis ccd8 mutants also show strong chlorosis when grown on medium with moderate levels of iron. This previously unappreciated role may have implications for the use of strigolactones in agricultural contexts.</p>
</div>
</div>
</div>
</section>
<section id="t28-upper-level-and-cross-hierarchical-regulation-of-predominantly-expressed-phenolic-genes-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t28-upper-level-and-cross-hierarchical-regulation-of-predominantly-expressed-phenolic-genes-in-maize">T28: Upper level and cross hierarchical regulation of predominantly expressed phenolic genes in maize</h2>
<p><font color="#336699">Biochemical and Molecular Genetics </font> <a href="mailto:Ankita.Abnave@rockets.utoledo.">Ankita Abnave</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-55-contents" aria-controls="callout-55" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-55" class="callout-55-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p><a href="mailto:Ankita.Abnave@rockets.utoledo.edu">Abnave, Ankita</a><sup>1</sup><br><a href="mailto:jerrin.john@rockets.utoledo.edu">John, Jerrin</a><sup>2</sup><br><a href="mailto:doseffan@msu.edu">Doseff, Andrea I,</a><sup>3</sup><br><a href="mailto:grotewol@msu.edu">Grotewold, Erich</a><sup>4</sup><br><a href="mailto:john.gray5@utoledo.edu">Gray, John</a><sup>1</sup><br></p>
<p><sup>1</sup>Department of Biological Sciences, University of Toledo, Toledo, OH 43606, USA<br><sup>2</sup>Department of Biology, Loyola University, Chicago, IL 60660, USA<br><sup>3</sup>Department of Physiology, and Department of Pharmacology and Toxicology, Michigan State University, East Lansing, MI 48824, USA<br><sup>4</sup>Department of Biochemistry and Molecular Biology, Michigan State University, East Lansing, MI 48824, USA<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-56-contents" aria-controls="callout-56" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-56" class="callout-56-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>There is strong interest in deciphering the gene regulatory networks (GRNs) that govern plant specialized metabolism to assist in plant breeding. Here, we investigated the GRN governing phenolic biosynthesis pathways from which ~ 8000 plant specialized metabolites are derived. Previously it was established that 19 predominantly expressed maize phenolic (PEP) genes are sufficient to explain &gt;70% of the metabolic flux through the phenylpropanoid, monolignol, and flavonoid branches of this pathway. A yeast-1-hybrid (Y1H) gene centric screening approach was employed to discover upper level (tiers 2, 3, and 4) regulators of maize PEP genes. These regulators were examined by co-expression analyses, and a subset of protein-DNA interactions (PDIs) validated in vivo1 by ChIP-qPCR and luciferase reporter assays in maize protoplasts. We describe a comprehensive GRN composed of 429 PDIs that exhibits hubs with high connectivity and cross hierarchical regulation of PEP genes in different branches of the pathway. The core PEP GRN is composed of 21,16, and 1 transcription factors (TFs) that operate as tier 2, 3, and 4 regulators respectively. Of these, ZmC3H42 regulates tier 1, 2 and 3 genes and also exhibits autoregulation. The core GRN includes TFs and TFs that are conserved in other plant species and that are implicated in phenolic gene regulation (e.g., ZmMYB40/53/100, ZmMADS9, and ZmWD40.1/PAC1). The GRN also includes conserved TFs (e.g., ZmC3H9, ZmHB20/79, ZmNAC103/123, ZmMYB19/26, ZmMYBR87, ZmDOF3, ZmbZIP67, ZmTCP30, and ZmbHLH128) which indicate that maize PEP genes are under developmental regulation and also fall under the control of biotic and abiotic stress. Together, the PEP GRN describes a complex regulatory mechanism that has evolved to coordinately regulate many phenolic genes in response to multiple internal and external signals and can guide efforts aimed at manipulating phenolic levels in plants.</p>
</div>
</div>
</div>
</section>
<section id="t29-elucidating-the-metabolomic-and-single-cell-transcriptomic-landscape-of-maize-leaf-senescence" class="level2">
<h2 class="anchored" data-anchor-id="t29-elucidating-the-metabolomic-and-single-cell-transcriptomic-landscape-of-maize-leaf-senescence">T29: Elucidating the metabolomic and single-cell transcriptomic landscape of maize leaf senescence</h2>
<p><font color="#336699">Biochemical and Molecular Genetics </font> <a href="mailto:MBRAR@CLEMSON.EDU">MANWINDER SINGH BRAR</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-57-contents" aria-controls="callout-57" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-57" class="callout-57-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Brar, Manwinder Singh<sup>1</sup><br>Kumar, Rohit<sup>1</sup><br>Ballmant, Kelly<sup>2</sup><br>Tharayil, Nishanth<sup>3</sup><br>McMahan, Christopher<sup>4</sup><br>Sekhon, Rajandeep S.<sup>1</sup><br></p>
<p><sup>1</sup>Department of Genetics and Biochemistry, Clemson University, Clemson, SC, USA 29634<br><sup>2</sup>Horticultural Science Department, University of Florida, Gainesville, FL, USA 32611<br><sup>3</sup>Department of Plant and Environmental Sciences, Clemson University, Clemson, SC, USA 29634<br><sup>4</sup>School of Mathematical and Statistical Sciences, Clemson University, Clemson, SC, USA 29634<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-58-contents" aria-controls="callout-58" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-58" class="callout-58-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Delayed leaf senescence (a.k.a. staygreen) is associated with higher grain yield, increased total dry matter, and heat and drought tolerance. To resolve the genetic architecture of senescence, we generated a high-resolution map of physiological, transcriptomic, and metabolic processes underlying natural diversity for staygreen. Analysis of distinct physiological indicators of senescence revealed that there are multiple paths to staygreen traits specified by temporally and spatially distinct cellular processes. Time-course analysis of primary and secondary leaf metabolome in 10 inbred lines, five staygreen and five non-staygreen, at three developmental stages revealed substantial metabolic variation for staygreen in maize. Combinatorial analysis of metabolic and phenotypic variation using marginal scan revealed striking differences in metabolome and found that 44 (92%) primary metabolites and 1201 (96%) secondary mass features underlie variation for the staygreen trait. In-depth analysis of flux in the secondary metabolism revealed that flavonoids (naringenin chalcone, kaempferol, and eriodictyol) are crucial for discerning the staygreen phenotype. Functional analysis of these metabolites in Arabidopsis validated the role of naringenin chalcone and eriodictyol in delaying senescence. To enhance the resolution of transcriptional changes during senescence, we compared the single-cell transcriptome of a staygreen and a non-staygreen inbred lines and identified that bundle sheath or mesophyll cells regulate transcriptional activity. Remarkably, we identified 2 and 7 distinct clusters of bundle sheath and mesophyll cells, respectively, indicating transcriptional divergence within these cell types during senescence. Our findings will identify novel components of biological organization that regulate staygreen in maize.</p>
</div>
</div>
</div>
</section>
<section id="t30-potent-pollen-gene-regulation-by-dna-glycosylases-in-maize" class="level2">
<h2 class="anchored" data-anchor-id="t30-potent-pollen-gene-regulation-by-dna-glycosylases-in-maize">T30: Potent pollen gene regulation by DNA glycosylases in maize</h2>
<p><font color="#0066CC">Transposons &amp; Epigenetics </font> <a href="mailto:gent@uga.edu">Jonathan Gent</a></p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-59-contents" aria-controls="callout-59" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Author List
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-59" class="callout-59-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Zeng, Yibing<sup>1</sup><br>Somers, Julian<sup>1</sup><br>Bell, Harrison<sup>2</sup><br>Dawe, R Kelly<sup>1 3</sup><br>Fowler, John<sup>2</sup><br>Nelms, Brad<sup>3</sup><br>Gent, Jonathan<sup>3</sup><br></p>
<p><sup>1</sup>Department of Genetics, University of Georgia; Athens, GA, USA 30602<br><sup>2</sup>Department of Botany and Plant Pathology, Oregon State University; Corvallis, Oregon, USA 97331<br><sup>3</sup>Department of Plant BIology, University of Georgia; Athens, GA, USA 30602<br></p>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-60-contents" aria-controls="callout-60" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Abstract
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-60" class="callout-60-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>While DNA methylation primarily represses transposable elements (TEs), some genes are repressed by DNA methylation in plant body tissues and require demethylation by DNA glycosylases (DNGs) for activation in endosperm and pollen. Activity of either one of two DNGs, MDR1 or DNG102, is essential for pollen viability in maize. Using single-pollen mRNA sequencing on pollen segregating mutations in both genes, we identified approximately 60 candidate DNG target genes. While they are silent or barely detectable in the plant body, this small set of genes accounts for over 10% of the pollen polyadenylated transcriptome. They are unusual in their tendency to lack introns but even more so in their having TE-like methylation in their coding DNA sequence. They are strongly enriched for genes that are predicted to function in cell wall loosening to support the rapid tip growth characteristic of pollen tubes as they carry the sperm cells through maternal apoplast and extracellular matrix of the pistil. These results suggest a function for DNA methylation and demethylation in regulating genes with potential for extremely high expression in pollen but constitutive silencing elsewhere.</p>
</div>
</div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
<p>Produced by <a href="mailto:hello@danielkick.com">Daniel Kick, PhD</a></p>
</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




</body></html>